packages/react-scripts/template-ef/.buildpacks

```diff
@@ -0,0 +1 @@
+https://github.com/heroku/heroku-buildpack-nodejs.git
```

Comment:
this is not sufficient

Reply:
what else is required for the buildpacks? Gabe kind of handed over this PR for me to "finish" and test.

Reply:
oh well it is for straight express, but not if we do Frontier